src/cryfs/filesystem/CryDevice.cpp

```diff
@@ -135,6 +135,7 @@ unique_ref<FsBlobRef> CryDevice::LoadBlob(const bf::path &path) {
 }
 
 void CryDevice::statfs(const bf::path &path, struct statvfs *fsstat) {
+  (void)path;
```

Comment:
We have a macro for this in the codebase. You can use `UNUSED(variable);` from https://github.com/cryfs/cryfs/blob/4069358b31b56f6c1fa91271aa6d20a211ab347a/src/cpp-utils/macros.h